discretisedfield/mesh.py

```diff
@@ -229,6 +238,38 @@ def __init__(
         if "isplane" not in attributes.keys():
             self.attributes["isplane"] = False
 
+    @property
+    def bc(self):
+        """Boundary condition for the mesh."""
+        return self._bc
+
+    @bc.setter
+    def bc(self, bc):
+        if bc not in {"neumann", "dirichlet", ""}:
+            for char in bc:
+                if char not in self.region.dims:
+                    raise ValueError(f"Axis {char} is absent in {self.region.dims}.")
```

Comment:
This is based on the assumption that `region.dims` can only have single-letter names (which is not required). We have to think about boundary conditions in more detail and probably need a different syntax. (This is also relevant for @samjrholt )

I would not change anything in this PR.

Reply:
I think this is good for now and we can think about `bc` in more detail in the future.

Reply:
Yes, good point. It would be good to discuss this,